android/src/main/java/com/reactnativegruveo/GruveoModule.java

```diff
@@ -145,6 +145,11 @@ public void recordingStateChanged(boolean us, boolean them) {
         sendEvent("recordingStateChanged", null);
     }
 
+    @Override
+    public void recordingFilename(String filename) {
+        sendEvent("recordingFilename", filename);
```

Comment:
Shouldn't `recordingFilename` also be added to: https://github.com/senseobservationsystems/react-native-gruveo/blob/master/index.d.ts#L36

and 

https://github.com/senseobservationsystems/react-native-gruveo/blob/master/index.js#L40

Reply:
Oh yes! I missed that.

Reply:
Just pushed my update for this.